### PR TITLE
Add fallbacks for venue commercial

### LINKED ISSUE #974: Venue Commercial->Picture->Icon degradation
On the splash page the Venue should display an Commercial or Picture or Icon. Depending on what is there. Currently it shows a Commercial or nothing.

### PR DESCRIPTION
**Pull request by arun1595**
Fixes #974

### CODE REVIEW

File: app/views/conference/_venue.html.haml [hunk 1]
```diff
@@ -11,7 +11,15 @@
           .thumbnail
             .flexvideo{ id: "resource-content-#{@conference.venue.commercial.id}"}
               = render partial: 'shared/media_item', locals: { commercial: @conference.venue.commercial }
-    
+      - elsif @conference.venue.picture.present?
+        .col-md-5
+          .thumbnail
+            = image_tag @conference.venue.picture.url, alt: "Venue", size: "560x315"
```
Comment (hennevogel) [inline app/views/conference/_venue.html.haml]: Please do not hardcode sizes. Use [bootstrap image classes](http://getbootstrap.com/css/#images)

File: app/assets/stylesheets/osem-splash.css.scss [hunk 2]
```diff
@@ -170,4 +170,19 @@
       color: white;
     }
   }
-}
\ No newline at end of file
+}
+
+/* Venue icon styles */
+.icon-container {
```
Comment (hennevogel) [inline app/assets/stylesheets/osem-splash.css.scss]: Please do not hardcode sizes. This needs to work on all screen sizes.

File: app/views/conference/_venue.html.haml (hunk 1 above)
Comment (arun1595) [inline app/views/conference/_venue.html.haml]: I hardcoded it to match the iframe width and height which we use to display the commercial. Okay, I will get remove the hardcoded values.

**Comment by arun1595** [comment]
@hennevogel The tests pass locally but travis is complaining about an error on a page which I didn't modify at all. Can you restart the build?

**Comment by hennevogel** [comment]
Merged it in a slightly different form :-)

Thank you @arun1595 :bouquet:

**Pull request merged**